docs/configuration/mongodb-event-flattening.asciidoc

```diff
@@ -177,9 +182,81 @@ Such document will be encoded as
 
 This option allows you to convert a hierarchical document into a flat structure suitable for a table-like storage.
 
+=== MongoDB `$unset` handling
+
+MongoDB allows you to make `$unset` operations which allows you to remove a certain field from a Document, and since the collections are schemaless it becomes hard to find a way to tell the consumers/sinkers which a field is now missing, the approach debezium uses is to set the desired to remove field to null value.
```

Comment:
s/debezium/Debezium/
```suggestion
MongoDB allows you to run `$unset` operations which allows you to remove a certain field from a document, and since the collections are schemaless, it becomes hard to find a way to tell the consumers/sinkers which a field is now missing, the approach Debezium uses is to set the desired to remove field to null value.
```